packages/common/src/components/popover/Popover.vue

```diff
@@ -15,15 +17,19 @@ import {
   onMounted,
   onUnmounted,
 } from '@vue/composition-api';
+import { createPopper } from '@popperjs/core';
 
 export default defineComponent({
   name: 'v-popover',
   setup() {
     const isOpen = ref(false);
     const root = ref<HTMLElement | null>(null);
+    let _popover = ref(null);
```

Comment:
Not a fan of underscore prefixes, what about:

```suggestion
    let popoverRef = ref(null);
```